docs/scos/dev/glue-api-guides/202212.0/decoupled-glue-api.md

```diff
@@ -0,0 +1,235 @@
+
+### Decoupling from conventions
+
+Old Glue API was tightly coupled with a JSON:API convention, and all resources have to follow it. With the current infrastructure, resources can use any implemented conventions, create new ones, or even not use any. In this case, the "no convention" approach is used, and a request and response is formatted as a plain JSON. For more details, see [Create and change Glue API conventions](/docs/scos/dev/glue-api-guides/{{page.version}}/create-and-change-glue-api-conventions.html).
+
+### New type of application: Glue Backend API application
+
+With the current setup out of the box, we have an additional Glue Backend API (BAPI) application that is meant to be an API application for our Back Office. This means that with this new application, infrastructure has direct access to Zed facades from BAPI resources. Also, out of the box, we have a separate `/token` resource specifically for BAPI that uses Back Office users' credentials to issue a token for BAPI resource.
+
+For more details about the difference between SAPI and BAPI, refere to [Backend and storefront API module differences](/docs/scos/dev/glue-api-guides/{{page.version}}/backend-and-storefront-api-module-differences.html).
+
+### Authentication servers
+Current infrastructure lets you switch between different authentication servers. For example, this can be useful if you want to use Auth0 or any other server in addition to implemented servers.
+
+For more details and examples, see [Use authentication servers with Glue API](docs/scos/dev/glue-api-guides/{{page.version}}/use-authentication-servers-with-glue-api.html).
+
+
+## Glue Storefront API
+
+![Glue Storefront API](/_drafts/glue-storefront-api.jpeg)
+
+## Glue Backend API
+
+![Glue Backend API](/_drafts/glue-backend-api.jpeg)
+
+## Infrastructure
+
+Decoupled Glue API infrastructure is implemented in the same layer of Spryker Commerce OS and called Glue, as the old one. It is responsible for providing API endpoints, processing requests, as well as for communication with other layers of the OS to retrieve the necessary information. Separate applications are implemented as separate modules—for example, `GlueStorefrontApiApplication`, and `GlueBackendApiApplication`. Each application has its own bootstrapping and a separate virtual host on the Spryker web server (Nginx by default).
+
+Logically, the Glue layer can be divided into separate parts:
+
+* **`GlueApplication` module**: The `GlueApplication` module provides a framework for constructing API resources and selecting a proper application. It intercepts all HTTP requests at resource URLs (for example, `http://mysprykershop.com/resource/1`), selects a proper application based on a bootstrap file, does content negotiation, and selects applicable convention if one exists, proxy request to the proper convention or uses default flow. Also, this module is used for the fallback to the old Glue API.
```

Comment:
“if one exists, proxy request to the proper convention or uses default flow” - this part is hard to understand. Do you mean “proxy requests”? If yes, then rephrase to something like “if one exists, proxy requests the proper convention or uses default flow”